### DISCUSSION TITLE
Missing sizes from output

### DISCUSSION THREAD

**Discussion by tmaly1980** [tailwindlabs/tailwindcss#10526] [Help]
I follow the instructions (https://tailwindcss.com/docs/installation) but seem to be missing some output sizes, ie -left-4, my-4, my-6, etc.... in my output.css . For the vertical margin, there is only my-8 and no other size at all. For right margin, there's only mr-3 and no others. There are no .ml- classes.

I'm also missing the shadow classes, probably others too.

Any idea how this could happen?

This is my tailwind.config.js:

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    './src/**/*.{html,js,vue}',
  ],
  theme: {
    extend: {},
  },
  plugins: [
    require('flowbite/plugin')
  ],
}
```

I have an input.css like this:

```
@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";
```

I run this:

` npx tailwindcss -i ./src/assets/tailwind/input.css -o ./src/assets/tailwind/output.css`

I've linked to my output.css here: https://pastebin.com/kXz3P46m

**Comment by wongjn** [comment]
Common reasons:

- Tailwind has not run a compilation since the classes were added.
- The classes were constructed with string interpolation/concatenation and do not appear as full, unbroken strings in the source code.
- The `content` file globs do not fully cover files that use the classes.
